osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -186,7 +186,7 @@ private WorkingBeatmap addDifficultyToSet(BeatmapSetInfo targetBeatmapSet, IBeat
             targetBeatmapSet.Beatmaps.Add(newBeatmap.BeatmapInfo);
             newBeatmap.BeatmapInfo.BeatmapSet = targetBeatmapSet;
 
-            Save(newBeatmap.BeatmapInfo, newBeatmap, beatmapSkin);
+            save(newBeatmap.BeatmapInfo, newBeatmap, beatmapSkin, false);
```

Comment:
If I must live with this then I would ask to at least do

```suggestion
            save(newBeatmap.BeatmapInfo, newBeatmap, beatmapSkin, transferCollections: false);
```

Yes I know rider will autohint this but let's at least try to be considerate towards everyone else. And maybe have this greppable in the future.